cert/build

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eufo pipefail
+set -efo pipefail
```

Comment:
is the 'no unset' rule removed here on purpose?

Reply:
Oh jeah, else the build script fails with usage of unbound variables (somehone an empty array is considered unbound)